backend/server.js

```diff
@@ -0,0 +1,38 @@
+import express from "express";
+import { sequelize } from "./database.js";
+import { Stock } from "./models/stock.js";
+import cors from "cors";
+import morgan from "morgan";
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(cors());
+app.use(express.json());
+app.use(morgan());
+
+app.get("/stocks/:ticker", async (req, res) => {
+    try {
+        const stock = await Stock.findOne({
```

Comment:
This is fine right now. But this code will grow, and it be beneficial to hold the queries and the business logic in a separate place that gets called from here.